docs/dev/troubleshooting/docs-contribution/docs.md

```diff
@@ -11,4 +11,10 @@ As collaboration on contribution to zkSync documentation is such an integral par
     - If you find a problem with a specific page in the zkSync documentation, you can always [edit the page](./edit-doc.md) to fix it.
     - If you want to discuss a larger problem or propose a major change to the zkSync documentation, you can raise an [issue on Github](https://github.com/matter-labs/zksync-web-v2-docs/issues).
```

Comment:
Can we create an issue template so we make sure people include all the information we'd need? There is some info on how to do it here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository